### PR TITLE
chore: doc review.

### PR DESCRIPTION
**Pull request by billy-the-fish**
Doc review on the Develop guide.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,179 +1,198 @@
...
 
 Best practice is to add new tests when you commit new functionality.
 
-## pgai Architecture
+## The pgai build architecture
 
-The pgai extension consists of SQL files as well as Python packages. SQL files 
-are maintained in the [./sql](./sql) directory. The Python packages are 
-maintained in the [./src](./src) directory.
+pgai consists of [SQL](#sql) scripts and [Python](#python) packages.
```
Comment (jgpruitt) [inline DEVELOPMENT.md:132]: Question: The "normal" workflow is to just run `make install` from inside the docker container. That "just works". The other make targets do more specific things for flexibility/speed, but `make install` does it all. Should we note that here?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,179 +1,198 @@
...
-version of the Python code is installed under `/usr/local/lib/pgai/<version>`.
+You install pgai into your developer environment using the `ai-<version from>-<version-to>.sql` script. The  
+installation workflow for the SQL scripts is:
+
+1. **Clean your environment**: run `make clean-sql` to delete `./sql/<version-from>-<current-version>.sql`.
+
+   You define `<current-version>` in [./sql/ai.control](./sql/ai.control).
+
+1. **Build pgai**: run `make build-sql` to compile idempotent and incremental scripts
+   into `./sql/<version-from>-<current-version>.sql`.
```
Comment (jgpruitt) [inline DEVELOPMENT.md:168]:
```suggestion
1. **Build pgai**: run `make build-sql` to compile idempotent and incremental scripts
   into `./sql/ai--*<current-version>.sql`.
```

**Review by jgpruitt — CHANGES_REQUESTED** [review]

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -1,179 +1,198 @@
...
+1. **Clean your environment**: run `make clean-py` to remove build artifacts from your developer environment.
+1. **Install pgai**:
+   to compile and install the python package with its associated dependencies.
+   * Current version: run `make install-py`.
+   * Previous versions: run `make install-prior-py`. **IAIN**: can i choose a version?
+1. **Uninstall pgai**: run `make uninstall-py` and delete `/usr/local/lib/pgai`.
 
-The SQL functions add the appropriate path dynamically when looking to
-import modules using [site.addsitedir()](https://docs.python.org/3/library/site.html#site.addsitedir)
+   **IAIN**: does this delete all versions or just the latest version?
```
[the remark this answers is not in the document]
Comment (billy-the-fish) [reply DEVELOPMENT.md]: Updated.

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -1,179 +1,198 @@
...
+* Install:
+   * [Python3](https://www.python.org/downloads/).
+   * [Docker](https://docs.docker.com/get-docker/).
+   * A Postgres client like [Psql](https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows/) or [PopSQL](https://docs.timescale.com/use-timescale/latest/popsql/).
+* Retrieve the API Keys for the LLM cloud providers you are working with.
+* Clone the pgai source in `<pgai_install_dir>`:
    ```bash
+   cd <pgai_install_dir>
    git clone git@github.com:timescale/pgai.git
    ```
```
[the remark this answers is not in the document]
Comment (billy-the-fish) [reply DEVELOPMENT.md]: Eyupppp

File: DEVELOPMENT.md (hunk 1 above)
Comment (billy-the-fish) [reply DEVELOPMENT.md:132]: Added a section.

File: DEVELOPMENT.md [hunk 5]
```diff
@@ -1,179 +1,198 @@
...
-In order to support multiple versions of the extension being 
-installed and upgraded from/to in a given postgres installation, we have to 
-install every version of our python code and its associated dependencies. Each 
-version of the Python code is installed under `/usr/local/lib/pgai/<version>`.
+You install pgai into your developer environment using the `ai-<version from>-<version-to>.sql` script. The  
+installation workflow for the SQL scripts is:
+
+1. **Clean your environment**: run `make clean-sql` to delete `./sql/<version-from>-<current-version>.sql`.
+
+   You define `<current-version>` in [./sql/ai.control](./sql/ai.control).
```
[the remark this answers is not in the document]
Comment (billy-the-fish) [reply DEVELOPMENT.md]: Updated

**Review by jgpruitt — CHANGES_REQUESTED** [review]

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**